src/olympia/conf/dev/settings.py

```diff
@@ -45,9 +45,7 @@
 INBOUND_EMAIL_DOMAIN = env('INBOUND_EMAIL_DOMAIN',
                            default='addons-dev.allizom.org')
 
-SYSLOG_TAG = "http_app_addons_dev"
-MOZLOG_NAME = SYSLOG_TAG
-SYSLOG_TAG2 = "http_app_addons_dev_timer"
+MOZLOG_NAME = "http_app_addons_dev"
 SYSLOG_CSP = "http_app_addons_dev_csp"
```

Comment:
never did work out if `SYSLOG_CSP` is used or what it is/was used for.

Reply:
We should talk with ops and remove it as well. Doesn't appear to be used at all.